src/astro/files/base.py

```diff
@@ -38,11 +40,11 @@ def __init__(
         )
 
     @property
-    def path(self) -> str:
+    def path(self) -> Any:
```

Comment:
Why do we need to change this?

Reply:
Surprisingly, I had not changed that file but its throwing the following mypy error:
```
src/astro/files/base.py: note: In member "path" of class "File":
src/astro/files/base.py:44: error: Returning Any from function declared to
return "str"  [no-any-return]
```

But the type of path is `str`

Reply:
The test passed when I reverted the changes @sunank200 in https://github.com/astronomer/astro-sdk/pull/740/commits/96a17d6bb035db24ee99d68bfbb09af9da01816a 

Where was it failing for you?

Reply:
It failed for me in local while doing the commit @kaxil